tasty-query/shared/src/main/scala/tastyquery/Symbols.scala

```diff
@@ -111,6 +113,14 @@ object Symbols {
         myPrivateWithin = privateWithin
         this
 
+    private[tastyquery] final def addAnnotations(annots: List[Annotation]): this.type =
+      myAnnotations = myAnnotations ::: annots
```

Comment:
this seems to allow setting multiple times, I suggest making the field nullable and then checking for null in the accessor